src/main/java/emissary/Emissary.java

```diff
@@ -1,21 +1,6 @@
 package emissary;
 
-import emissary.command.AgentsCommand;
-import emissary.command.Banner;
-import emissary.command.ConfigCommand;
-import emissary.command.DirectoryCommand;
-import emissary.command.EmissaryCommand;
-import emissary.command.EnvCommand;
-import emissary.command.FeedCommand;
-import emissary.command.HelpCommand;
-import emissary.command.PeersCommand;
-import emissary.command.PoolCommand;
-import emissary.command.RunCommand;
-import emissary.command.ServerCommand;
-import emissary.command.StopCommand;
-import emissary.command.TopologyCommand;
-import emissary.command.VersionCommand;
-import emissary.command.WhatCommand;
+import emissary.command.*;
```

Comment:
We prefer not to have star imports. Might need to adjust a setting in your IDE to avoid it auto converting to them